showcase/src/assets/translations/en.json

```diff
@@ -41,15 +41,21 @@
-          "LINKS": "For more details on how Angular Material deals with theming, check out <a href=\"https://material.angular.io/guide/theming\" target=\"_blank\">https://material.angular.io/guide/theming</a><br>You can find the Material Design color palettes here: <a href=\"https://material.io/design/color/the-color-system.html#tools-for-picking-colors\" target=\"_blank\">https://material.io/design/color/the-color-system.html#tools-for-picking-colors</a><br>If you want to create your own palette, this tool can help you: <a href=\"http://mcg.mbitson.com/\" target=\"_blank\">http://mcg.mbitson.com/</a>",
-          "TITLE": "Advanced usage with color palettes"
+        "ALT": {
+          "DETAILS": "In addition to the 3 basic palettes, Stark provides 3 alternative palettes: <code>$stark-alert-palette</code>, <code>$stark-grey-palette</code> and <code>$stark-success-palette</code>.",
+          "TITLE": "Alternative palettes"
         },
         "BASIC": {
-          "DETAILS": "All colors used in Stark can be set through a SCSS map called <code>$stark-color-theme</code>. Each color has a contrast property, which is needed to define the text color on top of this color. Once this map has been set, you need to import the Stark global theme to apply it.",
-          "EXAMPLE": "Here, a full version of the map is shown but you can also use it partially:",
-          "TITLE": "Basic usage"
+          "DETAILS": "Angular Material uses 3 color palettes: a primary palette, an accent palette and a warn palette. Stark gives you the ability to define those palettes through 3 SCSS maps: <code>$stark-primary-palette</code>, <code>$stark-accent-palette</code> and <code>$stark-warn-palette</code>.<br><br>For more details on how Angular Material deals with theming, check out <a href=\"https://material.angular.io/guide/theming\" target=\"_blank\">https://material.angular.io/guide/theming</a>",
+          "TITLE": "Basic palettes"
+        },
+        "CUSTOM": {
+          "DETAILS": "If you want to customize those palettes, you first need to create a palette and this tool is really usefull: <a href=\"http://mcg.mbitson.com/\" target=\"_blank\">http://mcg.mbitson.com/</a>. Just define your main color and it will generate the full palette, then you just need to copy the SCSS code and associate it to the palette you want to modify. Don't forget to import stark theming after the maps declaration. <br>Here, an example with default values on how to achieve this:",
```

Comment:
Typo: "usefull" -> "useful"

Can we have this text: "Don't forget to import stark theming after the maps declaration." in bold so it is noted as really important step?